.gitignore

```diff
@@ -4,4 +4,5 @@ ln1
 ln2
 it
 .idea
-.tmpenv
\ No newline at end of file
+.tmpenv
+.vscode
```

Comment:
I'm not a great fan of adding developer-specific things here. `.idea` kinda slipped through and should eventually be removed. Personal ignores can be put into `.git/info/exclude`.

Reply:
Good to know about `.git/info/exclude`! Adding this ignore felt odd